### PR TITLE
Fix[15661602]: user popover style issues

### PR DESCRIPTION
**Pull request by KornelKwiatkowski**
## Description

Task: https://tw.auroracreation.com/app/tasks/15661602

**Changes** 🏗

before:

image

image

after:

image

image

### CODE REVIEW

**Review by CzarekDryl — DISMISSED** [review]

File: src/components/v5/shared/UserPopover/partials/UserInfo.tsx [hunk 1]
```diff
@@ -66,79 +66,88 @@ const UserInfo: FC<UserInfoProps> = ({
...
-              'px-6': isTopContributorType,
-            })}
-            text={formatText({ id: 'userInfo.about.section' })}
-          />
-          <p
-            className={clsx('text-md text-gray-600', {
-              'px-6': isTopContributorType,
+      <div
+        className={clsx({
+          'pt-6': aboutDescriptionText || domains?.length,
```
Comment (bassgeta) [inline src/components/v5/shared/UserPopover/partials/UserInfo.tsx:71]: would it be possible for us to put the padding to the `aboutDescriptionText` and/or the domains section? so we don't need conditional classNames and we will just get spacing from those components?

**Review by rdig — CHANGES_REQUESTED** [review]
Top contributor background overlaps border

Screenshot from 2023-11-15 15-35-55
Screenshot from 2023-11-15 15-36-01
Screenshot from 2023-11-15 15-36-07

File: src/components/v5/shared/UserPopover/partials/UserInfo.tsx (hunk 1 above)
Comment (KornelKwiatkowski) [reply src/components/v5/shared/UserPopover/partials/UserInfo.tsx]: @bassgeta done :)

**Comment by KornelKwiatkowski** [comment]
@rdig could you verify it again? :)

**Review by TomaszChmielPagepro — DISMISSED** [review]

File: src/components/v5/shared/UserPopover/partials/UserInfo.tsx [hunk 2]
```diff
@@ -86,16 +86,16 @@ const UserInfo: FC<UserInfoProps> = ({
       )}
       {domains?.length ? (
         <div
-          className={clsx({
-            'px-6 pb-6 pt-2': isTopContributorType,
+          className={clsx('pt-6', {
+            'px-6 pb-6': isTopContributorType,
           })}
```
Comment (TomaszChmielPagepro) [inline src/components/v5/shared/UserPopover/partials/UserInfo.tsx:91]: What do you think about this notation?
```
className={clsx({
  'p-6': isTopContributorType,
  'pt-6': !isTopContributorType,
})}
```

**Comment by KornelKwiatkowski** [comment]
@melyndav I checked it, and add some fixes.

I think it looks fine now:

image

image

image

File: src/components/v5/shared/UserPopover/partials/UserInfo.tsx (hunk 2 above)
Comment (KornelKwiatkowski) [reply src/components/v5/shared/UserPopover/partials/UserInfo.tsx]: It's outdated after my last changes. Could you check this again? :)

**Review by TomaszChmielPagepro — APPROVED** [review]

**Review by rdig — CHANGES_REQUESTED** [review]
The "Top Contributor" popover is very broken for a user with reputation in multiple domains.

Also, the "About" section margins are wrong

Screenshot from 2023-11-17 14-20-41

However, all the other popovers for contributors seem to be rendering just fine

Screenshot from 2023-11-17 14-14-26

**Review by melyndav — CHANGES_REQUESTED** [review]
Hi @KornelKwiatkowski - thanks for your work again. 

image

A few more issues as shown in the screenshot.

1. The bottom HR shouldn't go full width.
2. The avatars should be 60x60 not 70x70.
3. The top left and right sections of the purple is outside of the container. 
4. The size of the font inside 'root' or the team doesn't match Figma.

**Comment by KornelKwiatkowski** [comment]
@melyndav

_"1. The bottom HR shouldn't go full width."_ - Are you sure you have the latest changes? 

_"2. The avatars should be 60x60 not 70x70."_ -  We have some inconsistencies here.

We followed the designs that came with the task: https://www.figma.com/file/5V8pr7iMwXsT9L3VAZsmUt/Colony-V5?type=design&node-id=9873-269126&mode=design&t=ttnAB0VtRiLvZdtk-4

I also see this design on Figma. I understand that we should follow this example, right? - https://www.figma.com/file/5V8pr7iMwXsT9L3VAZsmUt/Colony-V5?type=design&node-id=9771-344178&mode=design&t=ttnAB0VtRiLvZdtk-4

_"3. The top left and right sections of the purple is outside of the container."_ - As I wrote above, it's not related with this task. I fixed it here: https://github.com/JoinColony/colonyCDapp/pull/1314

_"4. The size of the font inside 'root' or the team doesn't match Figma."_ - fixed :)

**Review by rdig — CHANGES_REQUESTED** [review]
Bottom border for the top contributor is still full width

Screenshot from 2023-11-20 16-54-10

It's fine for the other contributors

Screenshot from 2023-11-20 16-54-18

Also, there's still the remaining issue of the tags rendering outside the popover

**Comment by KornelKwiatkowski** [comment]
Hi @rdig

_"Bottom border for the top contributor is still full width"_ - Fixed, thanks! :)

_"Also, there's still the remaining issue of the tags rendering outside the popover"_ - There is already a separate task for this

here: https://tw.auroracreation.com/app/tasks/15671662

**Review by melyndav — APPROVED** [review]
This is good to go, thanks @KornelKwiatkowski

**Review by rdig — APPROVED** [review]

**Review by TomaszChmielPagepro — APPROVED** [review]

**Pull request merged**